liquibase-core/src/main/java/liquibase/util/SqlUtil.java

```diff
@@ -341,4 +346,22 @@ public static String replacePredicatePlaceholders(Database database, String pred
         matcher.appendTail(sb);
         return sb.toString();
     }
+
+    /**
+     * Get the string sql statements from a given SqlStatement
+     * @param statement the statement to stringify
+     * @param sqlGeneratorFactory the SqlGeneratorFactory instance to use to generate the sql
+     * @param database the database to generate sql against
+     * @return the sql string or an empty string if there are no statements to generate
+     */
+    public static String getSqlString(SqlStatement statement, SqlGeneratorFactory sqlGeneratorFactory, Database database) {
+        Sql[] sqlStatements = sqlGeneratorFactory.generateSql(statement, database);
+        if (sqlStatements != null) {
+            return Arrays.stream(sqlStatements)
+                    .map(sql -> sql.toSql() + sql.getEndDelimiter())
+                    .collect(Collectors.joining("\n"));
```

Comment:
Should this use System.lineseparator? Maybe it is best to be consistent with the line ending.